python/marvin/tools/map.py

```diff
@@ -386,3 +417,257 @@ def plot(self, array='value', xlim=None, ylim=None, zlim=None,
             return (ax, fig)
         else:
             return ax
+
+    def _make_image(self, snr_min, log_cb, use_mask):
+        """Make masked array of image.
+
+        Args:
```

Comment:
ooph.  I can't make any promises.   at least I haven't been changing anyone's double quotes to single quotes.